tests/system_tests/psyclone/test_psyclone_system_test.py

```diff
@@ -160,17 +160,19 @@ def test_run(self, config):
             config.build_output / 'algorithm/algorithm_mod_psy.f90',
 
             # Expect these prebuild files
-            # todo: the kernal hash differs between fpp and cpp, perhaps just use wildcards.
-            config.prebuild_folder / 'algorithm_mod.1602753696.an',  # x90 analysis result
-            config.prebuild_folder / 'my_kernel_mod.4187107526.an',  # kernel analysis results
-            config.prebuild_folder / 'algorithm_mod.5088673431.f90',  # prebuild
-            config.prebuild_folder / 'algorithm_mod_psy.5088673431.f90',  # prebuild
+            # The kernel hash differs between fpp and cpp, so just use wildcards.
+            config.prebuild_folder / 'algorithm_mod.*.an',  # x90 analysis result
+            config.prebuild_folder / 'my_kernel_mod.*.an',  # kernel analysis results
+            config.prebuild_folder / 'algorithm_mod.*.f90',  # prebuild
+            config.prebuild_folder / 'algorithm_mod_psy.*.f90',  # prebuild
         ]
 
-        assert all(not f.exists() for f in expect_files)
+        # Glob returns a generator, which can't simply be tested if it's empty.
+        # So use a list instead:
+        assert all(list(f.parent.glob(f.name)) == [] for f in expect_files)
```

Comment:
An alternative solution would be to split the expected results by directory.

```
expected_prebuild_files = ['algorithm_mod.*.an', 'my_kernel_mod.*.an', ...]
assert all(list(config.prebuild_folder.glob(pattern)) == [] for pattern in expected_prebuild_files

expected_build_files = ['algorithm/algorithm_mod.f90', ...
```

This might be neater. See what you think.

Reply:
Hi Matthew,

I am back from holiday. Thanks for the suggestion. I have implemented this in the newest commit. Now it is ready for review.

Best regards,
Jason